editor/test/integration/model_scene_test.clj

```diff
@@ -0,0 +1,50 @@
+;; Copyright 2020 The Defold Foundation
+;; Licensed under the Defold License version 1.0 (the "License"); you may not use
+;; this file except in compliance with the License.
+;; 
+;; You may obtain a copy of the License, together with FAQs at
+;; https://www.defold.com/license
+;; 
+;; Unless required by applicable law or agreed to in writing, software distributed
+;; under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+;; CONDITIONS OF ANY KIND, either express or implied. See the License for the
+;; specific language governing permissions and limitations under the License.
+
+
+(ns integration.model-scene-test
+  (:require [clojure.test :refer :all]
+            [dynamo.graph :as g]
+            [integration.test-util :as test-util]
+            [editor.model-scene :as model-scene]
+            [editor.math :as math]
+            [editor.types :as types])
+  (:import [javax.vecmath Point3d]))
+
+
+(deftest aabb
+  (test-util/with-loaded-project
+    (let [node-id (test-util/resource-node project "/mesh/test.dae")
+          aabb (g/node-value node-id :aabb)
+          min ^Point3d (types/min-p aabb)
+          max ^Point3d (types/max-p aabb)
+          dist (.distance max min)] ; distance in meters (converted from centimeters in the loader)
+      (is (and (> 20 dist) (< 10 dist))))))
```

Comment:
Fun fact: `<` and its kind can take multiple arguments. You can think of it as "Is each successive argument larger than the last?", so you can write this as:
```
(< 10 dist 20)
```
Neatly expressing that `dist` should be between those values.